docs/stdlib/constraints.rst

```diff
@@ -71,9 +67,7 @@ Constraints
 
```

Comment:
For this and `min_ex_value`, I'd like to, under the code example, further drive home the difference in these and their non-ex variants. Something like this:

```suggestion
    .. note::
    
    In the example above, in contrast to the ``max_value`` constraint, a value of
    the ``maxex_100`` type cannot be ``100`` since the valid range of
    ``max_ex_value`` does not include the value specified in the constraint.

```